src/plugins/validation/validator.js

```diff
@@ -0,0 +1,54 @@
+import PluginComponent from '../plugin.component';
+import LIVR from 'livr';
+import * as validationService from './validation.service';
+
+LIVR.Validator.defaultAutoTrim(true);
+
+const Plugin = PluginComponent('validator');
+
+class Validator extends Plugin {
+	constructor() {
+		super(...arguments);
+	}
+
+	onInit() {
+		const validationSettings = this.validationSettings;
```

Comment:
i think it will be a little bit more readable to use service here explicitly and remove validationSettings

Reply:
I just thought that in the case of getter we could use these setting inside of other methods (possible future)